### PR TITLE
Allow sending some trace to seq

### PR DESCRIPTION
**Pull request by asdacap**
- Created an 'all' target group in NLog, which include seq.
- In NLogConfigurator, only replace loglevel if pattern == '*'.
- This mean we can specify specific rule to have trace/debug log and publish them to seq.
- Also cleanup NLog.config a bit.

## Changes:
- Add `all` target group in NLog.
- Add rule to only replace seq rule with pattern == '*'.
- Add logic to disable seq if MinLevel is off.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [X] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [X] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe): 

## Testing
**Requires testing**

- [X] Yes
- [ ] No

**In case you checked yes, did you write tests??**

- [ ] Yes
- [X] No

**Comments about testing , should you have some** (optional)

- Tested to be able to specify multisyncmodeselector trace and it appear in seq.

## Further comments (optional)

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: src/Nethermind/Nethermind.Runner/Logging/NLogConfigurator.cs [hunk 1]
```diff
@@ -44,7 +44,7 @@ public static class NLogConfigurator
                         {
                             foreach (Target? ruleTarget in rule.Targets)
                             {
-                                if (ruleTarget.Name == "seq")
+                                if (ruleTarget.Name == "seq" && rule.LoggerNamePattern == "*")
```
Comment (LukaszRozmej) [inline src/Nethermind/Nethermind.Runner/Logging/NLogConfigurator.cs:47]: why is this change needed?

**Review by LukaszRozmej — APPROVED** [review]

File: src/Nethermind/Nethermind.Runner/Logging/NLogConfigurator.cs (hunk 1 above)
Comment (asdacap) [reply src/Nethermind/Nethermind.Runner/Logging/NLogConfigurator.cs:47]: If the user create a rule to send a specific pattern to just seq. Then this will replace it's log level unintentially.
